libs/@types/kiosk-browser/kiosk-browser.d.ts

```diff
@@ -4,10 +4,31 @@ declare namespace KioskBrowser {
     level: number // Number between 0–1
   }
 
+  export type PrintSides =
```

Comment:
confused about how these changes relate to this PR

Reply:
Yeah, I went down this rabbit hole because of my work on #349, so this snuck in here after votingworks/kiosk-browser#73.